### ISSUE TITLE
Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by BanderaOUN** [LunaPixelStudios/Better-Minecraft#344]
### Describe the Bug

I can't install the game it says: "Object reference not set to an instance of an object" 

### Reproduction Steps

1.
2.
3.
...

### Operating System

Windows 10

### Minecraft Version

1.16

### Modloader

Forge

### Modpack Version

v56

### Optifine

I used Optifine

**Comment by tobiii125** [comment]
i have the same issue, but its not only your modpack its also other ones, pls help me if you know the problem

**Comment by Shadowzeny** [comment]
Getting the same bug with 1.16 and 1.18.1 and 1.18.2 packs though its not this pack alone, seems to be effecting several of the 'newer' versions, I was able to install things for 1.12.1 but not any higher.

Confirmed java version is 8 ((as I use JDK 18.0.1.1 and JRE 8*)) 
Can't download from CurseForge.com as the manifest doesn't generate properly. 
Server seems to be working fine but not ideal for single player setup. 

![image](https://user-images.githubusercontent.com/38971476/167296081-72bc54d5-9dd4-422c-a1cd-80e5235b020c.png)
